src/Writer/AbstractFileWriter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Thumbimage\Thumbimage\Writer;
+
+abstract class AbstractFileWriter
```

Comment:
Why is this class abstract? All its methods are implemented. Maybe you wanted to remain some methods without implementations and to move the implementation to some of the child classes?